src/pages/OnboardingAccounting/BaseOnboardingAccounting.tsx

```diff
@@ -0,0 +1,192 @@
+                    }
+                }
+                return {
+                    keyForList: connectionName,
+                    text,
+                    leftElement: (
+                        <Icon
+                            src={accountingIcon}
+                            width={variables.iconSizeExtraLarge}
+                            height={variables.iconSizeExtraLarge}
+                            additionalStyles={[StyleUtils.getAvatarBorderStyle(CONST.AVATAR_SIZE.DEFAULT, CONST.ICON_TYPE_AVATAR), styles.mr3]}
+                        />
+                    ),
+                    isSelected: userReportedIntegration === connectionName,
+                };
+            })
+            .filter((item): item is OnboardingListItem => !!item);
+        const noneAccountingOption: OnboardingListItem = {
+            keyForList: null,
+            text: translate('onboarding.accounting.noneOfAbove'),
+            leftElement: (
+                <Icon
+                    src={Expensicons.Clear}
+                    width={variables.iconSizeNormal}
+                    height={variables.iconSizeNormal}
+                    fill={theme.success}
+                    additionalStyles={[StyleUtils.getAvatarBorderStyle(CONST.AVATAR_SIZE.DEFAULT, CONST.ICON_TYPE_AVATAR), styles.mr3, styles.onboardingSmallIcon]}
+                />
+            ),
+            isSelected: userReportedIntegration === null,
+        };
+        return [...policyAccountingOptions, noneAccountingOption];
+    }, [StyleUtils, styles.mr3, styles.onboardingSmallIcon, theme.success, translate, userReportedIntegration]);
+
+    const footerContent = (
+        <>
+            {!!error && (
+                <FormHelpMessage
+                    style={[styles.ph1, styles.mb2]}
+                    isError
+                    message={error}
+                />
+            )}
+            <Button
+                success
+                large
+                text={translate('common.continue')}
+                onPress={() => {
+                    if (userReportedIntegration === undefined) {
+                        setError(translate('onboarding.errorSelection'));
+                        return;
+                    }
+
+                    if (!onboardingPurposeSelected) {
+                        return;
+                    }
+
+                    Report.completeOnboarding(
+                        onboardingPurposeSelected,
+                        CONST.ONBOARDING_MESSAGES[onboardingPurposeSelected],
+                        undefined,
+                        undefined,
+                        onboardingAdminsChatReportID ?? undefined,
+                        onboardingPolicyID,
+                        undefined,
+                        onboardingCompanySize,
+                        userReportedIntegration,
+                    );
+
+                    Welcome.setOnboardingAdminsChatReportID();
+                    Welcome.setOnboardingPolicyID();
```

Comment:
@nkdengineer Maybe I missed that but do you recall why we clear the onboarding admin report id and policy id?

Reply:
Ah yeah that makes sense! I will look into older PRs then. Thanks!